### ISSUE TITLE
test  6.1.10 Stress test - support 4k varchars failing (test/dmlReturning.js)

### ISSUE THREAD

**Issue by raulsalitrero** [oracle/node-oracledb#226]
## Test 6.1.10 Stress test - support 4k varchars failing (test/dmlReturning.js)

test case is returning undefined instead of 4k string,
if you change to 3999 it works as expected.

tested on windows x86 32bit using 
instant client 11.2.0.3 sdk
IO.js v2.5 (32bit)
windows 10 64bit
oracle database version 11gr2 ( 11.2.0.1.0 64bit - production linux)

*output:*

```
1) 6. dmlReturning.js 6.1 NUMBER & STRING driver data type 6.1.10 Stress test - support 4k varchars:
     Uncaught TypeError: Cannot read property 'length' of null
      at C:\nodedbtest\node_modules\oracledb\test\dmlReturning.js:339:35
```

if size is set to 3999 dmlReturning.js line:323 like:

```javascript
      /*** string length **/
      var size = 3999;
```

test passes

**Comment by cjbj** [comment]
Thanks @raulsalitrero. The detail in the report is much appreciated. We will look at this.

**Comment by raulsalitrero** [comment]
just Confirming, 
test now passes correectly, fix in commit a5975887dd86e05475b4e9328e39de65156b7200 works.
thanks a lot.

**Comment by cjbj** [comment]
@raulsalitrero Thanks for checking.
